tools/scrape/github.py

```diff
@@ -472,15 +474,21 @@ def _extract_bundle_repos_from_file(file_contents):
         file_contents: A string of the contents of the file to search through.
 
     Returns:
-        A tuple (Vundle repos, NeoBundle repos). Each element is a list of
-        tuples of the form (owner, repo_name) referencing a GitHub repo.
+        A tuple (Vundle repos, NeoBundle, VimPlug repos). Each element is a list
+        of tuples of the form (owner, repo_name) referencing a GitHub repo.
```

Comment:
We're not returning a tuple here... you changed this to return a dict. Mind updating the docstring here accordingly?